doc/OnlineDocs/explanation/modeling/dae.rst

```diff
@@ -35,8 +35,8 @@ Pyomo.DAE introduces three new modeling components to Pyomo:
 
 As will be shown later, differential equations can be declared using
 using these new modeling components along with the standard Pyomo
-:py:class:`Var <pyomo.environ.Var>` and
-:py:class:`Constraint <pyomo.environ.Constraint>` components.
+:py:class:`Var<pyomo.environ.Var>` and
+:py:class:`Constraint<pyomo.environ.Constraint>` components.
 
```

Comment:
Note that these links are broken, because we document Var, etc using their actual location (and not the environ alias).  You can make this work by using a generic "find this class somewhere" syntax: ```:py:class:`.Var` ```